src/services/sports/index.jsx

```
@@ -186,7 +187,7 @@ export const AddSports = ({ setAddModal, dispatch, setReload, categoryId }) => {
   );
 };
 
-export const SportsPreviewer = ({ image, isOpen, onClose, isTop = false }) => {
+export const SportsPreviewer = ({ image, isOpen, onClose }) => {
```

ESLint

[error] 'image' is missing in props validation

(react/prop-types)